lisp/ess-julia.el

```diff
@@ -39,224 +39,14 @@
 
 (require 'compile); for compilation-* below
 (require 'ess-utils)
+(require 'julia-mode)
```

Comment:
If you installed from MELPA then the test generation code will not work.  

Don't know how to properly go about it. Can you for now run `make julia-mode.el` from `ESS/lisp/`?

We can also do `(when (locate-library "julia-mode.el") ..)` for a conditional install.